packages/asc-core/src/components/Radio/RadioInput.ts

```diff
@@ -0,0 +1,30 @@
+import styled from '../../styled-components'
+
+type Props = {
+  id: string
+  name: string
+  label: string
+  value?: string
```

Comment:
Shouldn't a radio button always have a value?

Reply:
Maybe not if it's disabled?